seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -5,6 +5,36 @@
   "helpRoom": "Help",
   "challenges": [
     {
+      "id": "587d774b367417c4c2512a9b",
+      "title": "Introduction to the Basic HTML and HTML5 Challenges",
+      "description": [
+        [
+          "",
+          "",
+          "HTML, or HyperText Markup Language, is a markup language to describe the structure of a web page. It uses a special syntax of elements to organize and give information about your content to the browser. Elements usually have opening and closing tags that surround and give meaning to content. For example, there are different tag options to place around text to show whether it's a heading, a paragraph, or a list.<br><br>The HyperText part of HTML comes from the early days of the web. Pages usually contained static documents that connected to other documents with hypertext links. As web pages and web applications grow more complex, the W3 Consortium updates HTML. The latest version, or specification, is HTML5.<br><br>This section introduces how to use HTML elements to give structure and meaning to your web content.",
```

Comment:
Will do - I'll move the 2nd and 3rd paragraphs (the 3rd is only 1 sentence long) onto a second slide.